### ISSUE TITLE
Custom Start

### ISSUE THREAD

**Issue by Danimob** [ENDERZOMBI102/Retro-Station#4]
An opposite of the trigger end thing (thx for doing that btw.).  Where it is placed the player starts the map, instead of in the entrance section.  This would be good for making seamless transitions between chambers.  Sort of like the info_player_start entity in hammer (I think that is what it is called.)

**Comment by ENDERZOMBI102** [comment]
yeah that is its name, and yeah, its possible, but very glitchy, because two info_player_start would conflict with each other, and maybe can cause a compile error every time you try to compile the puzzle

**Comment by Flash4433** [comment]
That's a good idea, however, I don't know if it's possible. The only way to make an `info_player_start` not conflict is to make it `MASTER`, which means it's the most important one.

Now, I would have to check the entrance corridors to see if they already have `MASTER`. If they do, it won't be possible. If they don't, I'll also have to move the entrance trigger (the trigger that closes the entrance door, starts the voice lines, the droppers, the lights...). This entrance trigger might be created by BEE2 itself, so it also won't be possible.

I'll check it out, but I don't think it'll work.

**Comment by Danimob** [comment]
I know nothing about this, so sorry if what I am about to say is useless, but some possible ideas on how to do it:
Have it count as a special entrance corridor in compile settings.  (This might work with the trigger)
Write in the description of the item to make the user delete the door in the PTI themself

**Comment by ENDERZOMBI102** [comment]
its possible, the info_player_spawn doesn't have the master flag checked

**Comment by ENDERZOMBI102** [comment]
i've decompiled 3 maps, and of all three the flags weren't checked

**Comment by Flash4433** [comment]
Alright. What about the trigger that enables everything on the map? It enables the lights, droppers... It also closes the entry door.

**Comment by ENDERZOMBI102** [comment]
We can simply copypaste it

**Comment by Flash4433** [comment]
It should just be at the entry door. Let's just hope it executes some sort of script, so it's easily copyable...

**Comment by Flash4433** [comment]
Also, this should be in DecoSta :P is there a way to move issues?...
